src/libpmemfile-posix/mkdir.c

```diff
@@ -111,7 +111,7 @@ _pmemfile_mkdirat(PMEMfilepool *pfp, struct pmemfile_vinode *dir,
 
 	/* mkdir("/") */
 	if (namelen == 0) {
-		ASSERT(parent == pfp->root);
+		ASSERT(vinode_is_root(parent)); /* XXX */
```

Comment:
Why XXX?

Reply:
I wasn't sure if this is meant to assert that the parent is the default root (the path "/"), or any root.
There was no difference before...